app/namespace.go

```diff
@@ -1208,6 +1241,49 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 				},
 			},
 		},
+		{
+			Name:    "failover",
+			Usage:   "Failover a temporal namespace",
+			Aliases: []string{"fo"},
+			Flags: []cli.Flag{
+				RequestIDFlag,
+				&cli.StringFlag{
+					Name:     NamespaceFlagName,
+					Usage:    "The namespace hosted on temporal cloud",
+					Aliases:  []string{"n"},
+					Required: true,
+				},
+				&cli.StringFlag{
+					Name:     namespaceRegionFlagName,
+					Usage:    "The region to failover to",
+					Aliases:  []string{"r"},
+					Required: true,
+				},
+				&cli.StringFlag{
+					Name:  regionCloudProviderFlagName,
+					Usage: "The cloud provider of the region. Default: aws",
+					Value: "aws",
+				},
+			},
+			Action: func(ctx *cli.Context) error {
+				namespaceName := ctx.String(NamespaceFlagName)
+				region := ctx.String(namespaceRegionFlagName)
```

Comment:
can we instead query their replicas and present them with a list of regions to fali over to?

Reply:
We can update the behavior in the future.